Module/STIG/Convert/Functions.XccdfXml.ps1

```diff
@@ -58,7 +58,7 @@ function ConvertFrom-StigXccdf
     # Get the xml data from the file path provided.
     $stigBenchmarkXml = Get-StigXccdfBenchmarkContent -Path $path
 
-    # Global variable needed to distinguish between the IIS server and site stigs. Server Stig needs xIISLogging resource, Site Stig needs XWebsite
+    # Global variable needed to distinguish between the IIS server and site stigs. Server Stig needs xIISLogging resource, Site Stig needs xWebsite
     $global:stigTitle = $stigBenchmarkXml.title
 
```

Comment:
on line 363 can we remove $global:stigRuleGlobal = $stigRule.  This variable doesn't appear to be used anywhere, as we've decided to use $stigRule instead.